TLM/TLM/Resources/lang.txt

```diff
@@ -227,4 +225,4 @@ Remove_this_citizen Remove this citizen
 Show_error_message_if_a_mod_incompatibility_is_detected Show error message if a mod incompatibility is detected
 Remove_parked_vehicles Remove parked vehicles
 Node_is_level_crossing This junction is a level crossing.\nYou cannot disable traffic lights here.
-Experimental_features Experimental features
\ No newline at end of file
+Vehicles_may_turn_on_red Vehicles may turn at red traffic lights
```

Comment:
To make your life easier, new translations should always be placed after the last line. Try to synchronize those ugly translation files or invent a new system for internationalization :)

Reply:
I had a hard time coping with mixed up translation files. Especially when you only see that the number of lines are different...

Reply:
It might be worth checking out the localisation system that is implemented in the RealTime mod: https://github.com/dymanoid/SkyTools/tree/ad86fe14479d3ecd97c7a4cf3ab1dcbe5a425782/src/SkyTools/Localization